app/presenters/document/web_resource_presenter.rb

```diff
@@ -125,12 +125,24 @@ def technical_metadata
       }
     end
 
+    def is_avi
```

Comment:
Cyclomatic complexity for is_avi is too high. [8/6]<br>Assignment Branch Condition size for is_avi is too high. [17.46/15]<br>Perceived complexity for is_avi is too high. [8/7]<br>Rename `is_avi` to `avi?`.